cloudsync/oauth/oauth_config.py

```diff
@@ -109,7 +109,7 @@ def start_auth(self, auth_url, scope=None, **kwargs):
 
     def wait_auth(self, token_url, timeout=None, **kwargs):
```

Comment:
I think there was some confusion on the last commit in the PR. I think Ilya and I were inquiring about why the 'timeout' parameter in the function signature wasn't being used. IMO the timeout parameter should default to 60 and be passed into the OAuthToken call.

Reply:
Disregard. Just saw your comment in slack. I see the timeout parameter is global and has a different use case